### PR TITLE
Fix/TAO-10479/datatable action buttons are not clickable

### PR DESCRIPTION
**Pull request by andrey-ponamarev**
**Related to task:**  https://oat-sa.atlassian.net/browse/TAO-10479
- Task for update tao-core https://github.com/oat-sa/tao-core/pull/2651

**Description:** Cover the case when `actions` option is Object
**Unit tests cli:** `npm run test`

### CODE REVIEW

File: src/datatable.js [hunk 1]
```diff
@@ -411,9 +411,13 @@ const dataTable = {
 
         const attachActionListeners = actions => {
             // Attach a listener to every action button created
-            _.forEach(actions, function (action) {
-                const css = `.${action.id}`;
-                const handler = action.action;
+            _.forEach(actions, (action, name) => {
```
[the remark this answers is not in the document]
Comment (andrey-ponamarev) [reply src/datatable.js:414]: BTW this case is covered by unit test https://github.com/oat-sa/tao-core-ui-fe/pull/167/files/74f839acae1302aa398abfe8834775ab45e93a60#diff-0da2e222e8553c6604feb28786f841faR628-R636

**Review by bziondik — APPROVED** [review]
Please, update JSDoc. For now we have only 1 case 
`* @param {Function} options.actions.xxx - the callback function for items xxx, with a single parameter representing the identifier of the items.`
Code review only
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: src/datatable.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrey-ponamarev) [reply src/datatable.js:464]: Comments are added. Please, let me know if I can improve it further

**Review by oatymart — APPROVED** [review]
Documentation looks nice and based on the test case the refactor should be safe.

**Pull request merged**